src/Simplex_tree/example/simplex_tree_multi.cpp

```diff
@@ -0,0 +1,57 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       David Loiseaux
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#include <gudhi/Simplex_tree.h>
+#include <gudhi/One_critical_filtration.h>
+
+#include <iostream>
+#include <initializer_list>
+
+struct ST_MULTI {
+public:
+	typedef Gudhi::linear_indexing_tag Indexing_tag;
+	typedef int Vertex_handle;
+	typedef float value_type;
+	using Filtration_value = Gudhi::multi_filtration::One_critical_filtration<value_type>;
+	typedef std::uint32_t Simplex_key;
+	static const bool store_key = true;
+	static const bool store_filtration = true;
+	static const bool contiguous_vertices = false;
+	static const bool link_nodes_by_label = true;
+	static const bool stable_simplex_handles = false;
+	static const bool is_multi_parameter = true;
+};
+
+using ST = Gudhi::Simplex_tree<ST_MULTI>;
+
+
+int main() {
```

Comment:
Shouldn't the example be added in `src/common/doc/examples.h` or is it only useful for examples appearing in the introduction?

Reply:
I added the example in the documentation on [ffaf004](https://github.com/GUDHI/gudhi-devel/pull/1123/commits/ffaf00448f552cb50eacf94422c88b80a9e595f8)